mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/91 help and lab buttons lookandfeel

to test:

- make sure help and lab buttons work as expected and match the ui mockups
- check the font size in the funds and bonus boxes. they should be larger and bolder to match the mockups
